src/Discord.Net.Commands/CommandServiceConfig.cs

```diff
@@ -15,5 +15,8 @@ public class CommandServiceConfig
 
         /// <summary> Determines whether RunMode.Sync commands should push exceptions up to the caller. </summary>
         public bool ThrowOnError { get; set; } = true;
+
+        /// <summary> Determines whether the command execution should fail if too many parameters are provided. </summary>
+        public bool FailOnTooManyArgs { get; set; } = true;
```

Comment:
I think a better name for this property would be `IgnoreExtraArguments`, and its behaviour should be inversed to match the opposite behaviour.